e2e/helpers/wallet.helper.ts

```diff
@@ -65,7 +67,12 @@ export class WalletHelper {
         return blockhash;
     }
 
-    async addAmountToAddress(payment: Payment, amount): Promise<UTXO> {
+    async addAmountToAddress(
+        payment: Payment,
+        amount: number,
+        addressType: 'p2wpkh' | 'p2tr' | 'p2pkh' | 'p2sh-p2wpkh',
```

Comment:
Kindly create a new type for this,  type AddressType = 'p2wpkh' | 'p2tr' | 'p2pkh' | 'p2sh-p2wpkh';